replica/database.hh

```diff
@@ -454,7 +454,7 @@ private:
     db::rate_limiter::label _rate_limiter_label_for_reads;
 
     void set_metrics();
-    seastar::metrics::metric_groups _metrics;
+    std::unique_ptr<seastar::metrics::metric_groups> _metrics = std::make_unique<seastar::metrics::metric_groups>();
```

Comment:
There's metrics_group::clear(), so no need to the additional indirection. I expect the same holds for view::stats.

Reply:
Right, thank you. Changed as suggested.